package.json

```diff
@@ -2,14 +2,22 @@
   "name": "zeppelin-element-library",
   "version": "1.2.1",
   "description": "Zeppelin Elements Library is an element library for digital Zeppelin products.",
-  "main": "bundle/zeppelin-element-library.cjs.js",
-  "module": "bundle/zeppelin-element-library.esm.js",
   "author": "Joerg Schneider <joerg.schneider@z-lab.com>",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/zepdev/zeppelin-element-library.git"
   },
+  "main": "bundle/zeppelin-element-library.cjs.js",
```

Comment:
all the bundled JavaScript files have to be named zel... as well like you do in the rollup bundle script